lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -61,7 +64,7 @@ class ReportRenderer {
     container.appendChild(this._renderReport(clone));
 
     // put the UIStrings back into original state
-    ReportRenderer.updateAllUIStrings(clonedStrings);
+    ReportRenderer.updateAllUIStrings(originalUIStrings);
```

Comment:
updated because it was a little confusing with `clone` being the cloned LHR, while `clonedStrings` was the clone of the original `Util.UIStrings`, *not* the translated strings found in `clone` :)